
public class ExceptionCentral 
{

	public static void main(String[] args) 
	{
		int[] numbers = {1,3,5,7};
		ArrayIndexOutOfBoundsException(numbers);
		
		
	}

	public void ArithmeticException()
	{
		
	}
	
	public void NullPointerException()
	{
		
		
		
		
		
	}
	
	public static void ArrayIndexOutOfBoundsException(int[] numbers)
	{
		int sum = 0;
		
		if (numbers== null)
		{
			System.out.println("The input to add array was a null array");
		}
		
		for (int i = 0; i <=numbers.length; i++ )
		{
			sum = sum + numbers [i];
		}
		
		System.out.println("Sum is " + sum);
	}
	
}
